solution2.py

```diff
@@ -0,0 +1,28 @@
+# https://leetcode.com/problems/contiguous-array
+# // Time Complexity : o(n)
+# // Space Complexity : o(n)
+# // Did this code successfully run on Leetcode : yes
+# // Any problem you faced while coding this : None
+#
+
+class Solution:
+    def findMaxLength(self, nums: List[int]) -> int:
```

Comment:
Please try the solution with a single hashmap and no other Data Structure as well.